Unknown number 2
Unknown [['Folder_test/taskParameters_processor_2_2394659579230.txt', 'TASK T0 can schedule\nTASK T1 can schedule\nTASK T8\t[60]\t1\t[0.522712227631]\t[0.008711870460516666]\nTASK T2 can schedule\nTASK T8\t[60]\t1\t[0.522712227631]\t[0.008711870460516666]\nTASK T3 can schedule\nTASK T8\t[60]\t1\t[0.522712227631]\t[0.008711870460516666]\nTASK T10\t[40]\t1\t[2.0957806395]\t[0.0523945159875]\nTASK T9\t[10]\t1\t[2.85810072495]\t[0.285810072495]\nTASK T4 can schedule\nTASK T8\t[60]\t1\t[0.522712227631]\t[0.008711870460516666]\nTASK T5 can schedule\nTASK T6 can schedule\nTASK T8\t[60]\t1\t[0.522712227631]\t[0.008711870460516666]\nTASK T10\t[40]\t1\t[2.0957806395]\t[0.0523945159875]\nTASK T9\t[10]\t1\t[2.85810072495]\t[0.285810072495]\nTASK T7 can schedule\nTASK T8\t[60]\t1\t[0.522712227631]\t[0.008711870460516666]\nTASK T10\t[40]\t1\t[2.0957806395]\t[0.0523945159875]\nTASK T9\t[10]\t1\t[2.85810072495]\t[0.285810072495]\n  Evaluated 110 individuals\n-- Generation 0 --\nTraceback (most recent call last):\n  File "mxc_run_ga.py", line 137, in <module>\n    main()\n  File "mxc_run_ga.py", line 102, in main\n    fitnesses = map(toolbox.evaluate, invalid_ind)\n  File "/home/romain/Projets/mxc_run_GA/mxc_run_ga_utils.py", line 87, in evalAlloc\n    schedulability,task_utilization = schedulabilityFunctions.Check_schedulability_utilization(list_aggregated_modal_server, list_task)\nTypeError: \'bool\' object is not iterable\n'], ['Folder_test/task.txt', 'TASK T0 can schedule\nTASK T4\t[15]\t1\t[3.75]\t[0.25]\nTASK T3\t[8]\t1\t[3.2]\t[0.4]\nTASK T5\t[12]\t1\t[2.4]\t[0.19999999999999998]\nTASK T1 can schedule\nTASK T3\t[8]\t1\t[3.2]\t[0.4]\nTASK T5\t[12]\t1\t[2.4]\t[0.19999999999999998]\nTASK T4\t[15]\t1\t[3.75]\t[0.25]\nTASK T2 can schedule\nTASK T3\t[8]\t1\t[3.2]\t[0.4]\nTASK T4\t[15]\t1\t[3.75]\t[0.25]\nTASK T5\t[12]\t1\t[2.4]\t[0.19999999999999998]\n  Evaluated 60 individuals\n-- Generation 0 --\nTraceback (most recent call last):\n  File "mxc_run_ga.py", line 137, in <module>\n    main()\n  File "mxc_run_ga.py", line 102, in main\n    fitnesses = map(toolbox.evaluate, invalid_ind)\n  File "/home/romain/Projets/mxc_run_GA/mxc_run_ga_utils.py", line 87, in evalAlloc\n    schedulability,task_utilization = schedulabilityFunctions.Check_schedulability_utilization(list_aggregated_modal_server, list_task)\nTypeError: \'bool\' object is not iterable\n']]
DBF value greater than 32768 0
NO LO tasks or HI tasks 0
Average Execution Time 0.0
SBF value greater than 32768 0
Execution greater than 30 minutes 0
Execution greater than 30 minutes details []
Modal server capacity not big enough 0
Execution Time []
